release/templates/release.api.pom.tpl

```diff
@@ -7,6 +7,6 @@
 >
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.liferay.portal</groupId>
-	<artifactId>release.dxp.api</artifactId>
+	<artifactId>__LIFERAY_RELEASE_PRODUCT_NAME__</artifactId>
```

Comment:
@pgydev, can you please rename here `__LIFERAY_RELEASE_PRODUCT_NAME__` to `__ARTIFACT_ID__` instead?

After all, the information that will be added is the `artifactId` which has the product name as part of it only. Does it make sense to you?

Thanks